src/fontra/views/editor/visualization-layer-definitions.js

```diff
@@ -430,6 +435,10 @@ registerVisualizationLayerDefinition({
       return;
     }
 
+    const { backgroundImage: backgroundImageSelection } = parseSelection(
```

Comment:
You don't need to parse the selection. Just check whether "image/0" is in the selection set.

Reply:
Good point. I changed it to `if (model.selection.has("backgroundImage/0")) {` 
BTW: I named it `backgroundImage/0` and not `image/0` as previously, because I saw you named it `backgroundImage` everywhere. I hope this is in your interest.